### ISSUE TITLE
find_learning_rate does not work anymore

### ISSUE THREAD

**Issue by lucaventurini** [zalandoresearch/flair#1108]
**Describe the bug**
Before, I've run several experiments with which I could plot the optimal learning rate as in the tutorial. Now, after an upgrade of the codebase, I cannot anymore reproduce the same experiments.
What I find is similar to issue #986 , but in my case the learning rate is never updated (while in the above issue is updated at least at the first step).

**To Reproduce**
I've run the experiments with commit 24b72a6dfd1d0c2519794e1183cacc42c932c4f4  , and set a start lr of 1e-08 and an end lr of 1000, with 200 iterations. I get a tsv with 200 lines where the lr is always the same (slightly above the start lr).

**Expected behavior**
I previously run the same experiments with commit 6c27431ecc731446f12bc3205bf68d0a859b9a8d and they seemed to work as in the tutorial. The lr in the tsv spanned all the interval specified.
